pkg/tfgen/test_data/regress-minirandom-schema.json

```diff
@@ -67,7 +67,7 @@
           "additionalProperties": {
             "$ref": "pulumi.json#/Any"
           },
-          "description": "Arbitrary map of values that, when changed, will trigger recreation of resource. See [the main provider\ndocumentation](../index.html) for more information.\n",
+          "description": "Arbitrary map of values that, when changed, will trigger recreation of resource. See the main provider documentation for\nmore information.\n",
```

Comment:
This change is odd - is the link removal expected?
The newline in the middle certainly doesn't look intended.

Reply:
I believe the change is correct.

- We the remove relative links because the registry doesn't have the same structure as upstream's docs. This is intentional.
- The newline is present in the upstream doc. `reformatText` moved it, but it doesn't look like a negative change.